pkg/asset/machines/worker.go

```diff
@@ -475,6 +475,14 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 			mpool := defaultPowerVSMachinePoolPlatform()
 			mpool.Set(ic.Platform.PowerVS.DefaultMachinePlatform)
 			mpool.Set(pool.Platform.PowerVS)
+			// TODO: Temporary patch to bring up the worker nodes, later should be removed
+			hardcode := &powervstypes.MachinePool{
+				ServiceInstance: "e449d86e-c3a0-4c07-959e-8557fdf55482",
+				ImageID: "11b3470c-e747-4f92-ba70-428054ca4672",
+				KeyPairName: clusterID.InfraID+"-key",
+				NetworkIDs: []string{"daf2b616-542b-47ed-8cec-ceaec1e90f4d"},
+			}
+			mpool.Set(hardcode)
```

Comment:
for this to work you'll also need the corresponding values added to the machine pool here: https://github.com/openshift-powervs/installer/blob/69765177a3b7a61bf697dd0f3d1e60553dbeca6b/pkg/types/powervs/machinepools.go#L4

and to update the `Set` function there to pick these up

Reply:
true that,! I was hurry to see the workers spinning, that's the reason I didn't focus on fixing the masters :P

Reply:
fixed this too.